java/src/test/resources/testdata

```diff
@@ -0,0 +1 @@
+../../../../test/testdata/
```

Comment:
Oh, I see what you mean by by the symlink. I don't think that will work well for Windows.

An option is to do the linking using Ant (via antrun-plugin). Eg:

```xml
<execution>
    <id>symlink-testdata</id>
    <phase>generate-test-resources</phase>
    <configuration>
        <target>
            <mkdir dir="${project.basedir}/src/test/resources"/>
            <symlink link="${project.basedir}/src/test/resources/testdata"
                resource="${project.basedir}/../test/testdata"
                overwrite="true"/>
        </target>
    </configuration>
    <goals>
        <goal>run</goal>
    </goals>
</execution>
```

Another option might be to simply copy the files to `${project.build.testOutputDirectory}/testdata`. I don't think I have a strong preference; none of it makes me too happy TBH.